api/src/main/kotlin/com/openlattice/search/Auth0UserSearchFields.kt

```diff
@@ -0,0 +1,20 @@
+package com.openlattice.search
+
+import com.fasterxml.jackson.annotation.JsonProperty
+
+// https://auth0.com/docs/users/user-search/user-search-query-syntax
+data class Auth0UserSearchFields(
+    @JsonProperty val email: String?,
+    @JsonProperty val name: String?
+) {
+    init {
+        // TODO - support multiple fields and construct a valid Lucene query string to pass to Auth0
```

Comment:
Can you create a JIRA ticket and add it here?
ex: `TODO - JIRA-XXXX - `

Reply:
https://jira.openlattice.com/browse/LATTICE-2805